File: 1170.cpp
#include <stdio.h>
 
int main() {
	int entrada=0,i,dias=0;
	float qtdeDia=0,metSuprimento=0;
	
	scanf("%d",&entrada);
	
	for(i=0; i < entrada;i++){
		dias=0;
		scanf("%f",&qtdeDia);
		
		while(qtdeDia > 1){
			dias++;
			metSuprimento = qtdeDia/2;
			qtdeDia=metSuprimento;
		}
		
		printf("%d dias\n",dias,qtdeDia);
	}
    return 0;
}

File: 1170.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var entrada=0,i,dias=0, qtdeDia=0,metSuprimento=0;
	
	entrada=lines.shift().split(' ');
	
	for(i=0; i < entrada;i++){
		dias=0;
		qtdeDia=lines.shift().split(' ');
		
		while(qtdeDia > 1){
			dias= dias+1;
			metSuprimento = qtdeDia/2;
			qtdeDia=metSuprimento;
		}
		
		console.log(dias+" dias");
	}
